docs/src/userguide/extension_types.rst

```diff
@@ -479,6 +479,64 @@ when it is deleted:
     We don't have: ['camembert', 'cheddar']
     We don't have: []
 
+
+C methods
+=========
```

Comment:
The section C methods was moved before subclassing because we need to use `cdef/@cfunc` constructs.